symfony/framework-bundle/5.1/config/routes.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+use App\Kernel;
+use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
+
+return static function (RoutingConfigurator $routes, Kernel $kernel): void {
+
+//    $routes->add('index', '/')
+//        ->controller('App\Controller\DefaultController::index');
```

Comment:
Could we do something to be able to do:
```php
->controller(DefaultController::class, 'index')
```
or
```php
->controller([DefaultController::class, 'index'])
```

Reply:
1st proposal doesn't work, 2nd would, works for me.

Reply:
updated to using `[...::class, 'index']` without `use` statement, because this is a commented example.